modules/EvseV2G/EvseV2G.cpp

```diff
@@ -16,11 +17,13 @@ void log_handler(openssl::log_level_t level, const std::string& str) {
         // ignore debug logs
         break;
     case openssl::log_level_t::warning:
-        EVLOG_warning << str;
+        // avoid logging function and line number
+        BOOST_LOG_SEV(::global_logger::get(), ::Everest::Logging::warning) << str;
         break;
     case openssl::log_level_t::error:
     default:
-        EVLOG_error << str;
+        // avoid logging function and line number
+        BOOST_LOG_SEV(::global_logger::get(), ::Everest::Logging::error) << str;
```

Comment:
see other comment about direct access to boost log

Reply:
see other comment's response